pkg/runtime/conversion_test.go

```diff
@@ -55,6 +57,16 @@ func TestStringMapConversion(t *testing.T) {
 		errFn    func(error) bool
 		expected runtime.Object
 	}{
+		"array works": {
+			input: map[string][]string{
+				"String":    {"not_used"},
```

Comment:
Need to handle quotes, escape characters, and arguments with commas in them (and all three).  This won't be safe to allow unless you can enforce that someone can't do an argument injection attack on the `exec` command query.

I.e. consider all query values tainted, ensure that there can never be a command argument injection attack in this code path.  Which means client and server have to define escaping rules for commas and quotes.